tests/Integration/Foundation/Console/AboutCommandTest.php

```diff
@@ -40,4 +41,16 @@ public function testItCanDisplayAboutCommandAsJson()
             ], $output['drivers']);
         });
     }
+
+    #[WithEnv('VIEW_COMPILED_PATH', __DIR__.'/../../View/templates')]
```

Comment:
I don't really like making this test depend on templates from other tests, but also I'm not sure about adding custom fixtures here as compiled views is the main concern of this about console command. I would be glad if you could help with the testing approach here.